src/scripts/main.js

```diff
@@ -76,84 +82,111 @@ for (const el of form) {
 
 /* -----Show notification if form data is invalid and add new employee----- */
 
-const pushNotification = (employeeName, employeeAge) => {
-  const div = document.createElement('div');
-  let haveError = false;
-
+const pushNotification = (value, div) => {
   div.classList = `notification`;
 
-  if (employeeName && employeeName.length < 4) {
+  if (value === 'warning') {
     div.insertAdjacentHTML('afterbegin', `
-    <h2 class='title'>Error message</h2>
-    <p>Name can not be less then 4 chars</p>
+      <h2 class='title'>Warning message</h2>
+      <p>Invalid data type</p>
   `);
-    div.classList.add('error');
-
-    haveError = true;
+    div.classList.add('warning');
   }
 
-  if (employeeAge && (employeeAge < 18 || employeeAge > 90)) {
+  if (value === 'errorName') {
     div.insertAdjacentHTML('afterbegin', `
-    <h2 class='title'>Error message</h2>
-    <p>Age should be from 18 to 90 years old</p>
+      <h2 class='title'>Error message</h2>
+      <p>Name can not be less then 4 chars</p>
   `);
     div.classList.add('error');
+  }
 
-    haveError = true;
+  if (value === 'errorAge') {
+    div.insertAdjacentHTML('afterbegin', `
+      <h2 class='title'>Error message</h2>
+      <p>Age should be from 18 to 90 years old</p>
+  `);
+    div.classList.add('error');
   }
 
-  if (haveError === false) {
+  if (value === 'success') {
     div.insertAdjacentHTML('afterbegin', `
-    <h2 class='title'>Success message</h2>
-    <p>You have successfully added an employee</p>
+      <h2 class='title'>Success message</h2>
+      <p>You have successfully added an employee</p>
   `);
     div.classList.add('success');
   }
+};
 
-  body.append(div);
+function showNotification(element) {
+  body.append(element);
 
   setTimeout(() => {
-    div.remove();
+    element.remove();
   }, 2000);
-
-  return !haveError;
-};
+}
 
 form.addEventListener('click', (eventObj) => {
   eventObj.preventDefault();
 
   const target = eventObj.target.closest('button');
 
+  let hasError = false;
+
   if (!target) {
     return;
   }
 
-  pushNotification(form.elements.name.value, form.elements.age.value);
+  const div = document.createElement('div');
+
+  if (form.elements.name.value === ''
+  || form.elements.position.value === ''
```

Comment:
```suggestion
    || form.elements.position.value === ''
```
И все остальные строки в этом if сдвинь на два пункта вправо